### PR TITLE
fix(Requires): display values that are already present in the value even if they are not available

### LINKED ISSUE #1208: nonexisting dependent / depending transformations not visible
https://github.com/keboola/transformation-bundle/issues/84

### PR DESCRIPTION
**Pull request by ondrejhlavacek**
FIXES #1208, FIXES keboola/transformation-bundle#84

### CODE REVIEW

File: src/scripts/modules/transformations/react/pages/transformation-detail/Requires.jsx [hunk 1]
```diff
@@ -61,6 +68,24 @@ export default React.createClass({
...
+    var missing = _.filter(currentTransformation.get('requires').toArray(), function(possiblyMissingTransformationId) {
+      if (_.find(options, function(option) {
+        return option.value === possiblyMissingTransformationId;
+      })) {
+        return false;
+      }
+      return true;
+    });
+    // add them to options
+    for (var i = 0; i < missing.length; i++) {
```
Comment (ujovlado) [inline src/scripts/modules/transformations/react/pages/transformation-detail/Requires.jsx]: @ondrejhlavacek zbavil by som sa tohto "foru", a asi tie options zrobil pomocou `map` a nasledne `concat` k options?

File: src/scripts/modules/transformations/react/pages/transformation-detail/Requires.jsx [hunk 2]
```diff
@@ -51,8 +51,15 @@ export default React.createClass({
   },
 
   getSelectOptions: function(transformations, currentTransformation) {
-    return _.sortBy(_.map(_.filter(transformations.toArray(), function(transformation) {
-      return parseInt(transformation.get('phase'), 10) === parseInt(currentTransformation.get('phase'), 10) && transformation.get('backend') === currentTransformation.get('backend') && transformation.get('id') !== currentTransformation.get('id');
+    var options = _.sortBy(_.map(_.filter(transformations.toArray(), function(transformation) {
```
Comment (ujovlado) [inline src/scripts/modules/transformations/react/pages/transformation-detail/Requires.jsx]: @ondrejhlavacek len kozmeticke var->let/const

**Comment by ujovlado** [comment]
@ondrejhlavacek napadlo mi este jedno vylepsienie. Co keby sme userovi dali rovno vediet, ze to je zmazana transformacia?

screenshot_2017-10-18_17-00-38

**Comment by ondrejhlavacek** [comment]
:cool: zapracuju!

**Comment by ondrejhlavacek** [comment]
image

Co takhle?

**Comment by ondrejhlavacek** [comment]
A z druhý strany takhle

image

File: src/scripts/modules/transformations/react/pages/transformation-detail/Requires.jsx (hunk 1 above)
Comment (ondrejhlavacek) [reply src/scripts/modules/transformations/react/pages/transformation-detail/Requires.jsx]: jasny!

**Pull request merged**